templates/field/field--node--oe-authors--oe-publication--teaser.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Authors field template.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{%- include 'field--multiple-values--comma-separated.html.twig' -%}
```

Comment:
Should not be pipe separated? Some authors might have commas like in skos terms.